code/modules/antagonists/nations/factions/civil.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationserv
+	name = "Servicetopia"
```

Comment:
Not greytide? This could use a better name

Reply:
If assistants are split off, the name is fine, but if they remain, it does needs a better name